_data/chains/eip155-23294.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "Oasis Sapphire",
+  "chain": "Sapphire",
+  "icon": "oasis",
+  "rpc": ["https://sapphire.oasis.dev", "wss://sapphire.oasis.dev/ws"],
+  "faucets": [],
+  "nativeCurrency": {
+    "name": "Sapphire Rose",
+    "symbol": "ROSE",
+    "decimals": 18
+  },
+  "infoURL": "https://docs.oasis.io/dapp/sapphire",
+  "shortName": "sapphire",
+  "chainId": 23294,
+  "networkId": 23294,
+  "explorers": [
+    {
+      "name": "Oasis Sapphire Explorer",
+      "url": "https://explorer.sapphire.oasis.dev",
```

Comment:
is not reachable

Reply:
Yeah, the Mainnet deployment is scheduled for Q4 2022. Should we register the chain ID with another PR then or can this be combined in this single PR?

Reply:
you can add the explorer later and add status incubating - just cannot check the explorer EIP compatibility now

Reply:
Done.